src/InfoViz/Native/FieldSelector/index.js

```diff
@@ -38,8 +38,6 @@ function fieldSelector(publicAPI, model) {
 
     if (el) {
       d3.select(model.container)
-        .style('overflow-y', 'auto')
-        .style('overflow-x', 'hidden');
```

Comment:
Does the field selector still scroll properly after this is removed? 
Line 40 is now unnecessary.